Source/Csla/CriteriaBase.cs

```diff
@@ -19,6 +19,7 @@ namespace Csla
   /// derived in a business class. 
   /// </summary>
   [Serializable]
+  [Obsolete("Use normal .NET or CSLA types", false)]
```

Comment:
We should explain that more in detail.
"Use a .Net POCO with a custom serializer or switch to ReadOnlyBase with creation through a dataportal to get a working instance. We recommend the POCO serializer. For more information please check the upgrade guide to CSLA v9"